torch/csrc/jit/jit_log.cpp

```diff
@@ -53,8 +61,9 @@ static std::unordered_map<std::string, size_t> parseJITLogOption(
 }
 
 bool is_enabled(const char* cfname, JitLoggingLevels level) {
-  static const char* c_log_level = std::getenv("PYTORCH_JIT_LOG_LEVEL");
-  static const std::unordered_map<std::string, size_t> files_to_levels =
+  std::string log_levels = get_jit_logging_levels();
+  const char* c_log_level = log_levels.c_str();
```

Comment:
I'm a little bit concerned we are now doing option parsing for every call. I guess we could look into caching this stuff later if it proves to be a problem. fyi: @ZolotukhinM

Reply:
Yeah you are write, I missed another problem after taking a look at it. If a user changes the environment variable then the logging level won't change. To prevent it, I might need to anyways cache the running level and check if there is any change in the environment variable. (and always set the env variable when the set API is called)
I'll create another commit for this change if it looks fine

Reply:
Yeah, we need to be careful not to use these macros on hot paths